src/interfaces/fees.spec.ts

```diff
@@ -0,0 +1,94 @@
+import BigNumber from "bignumber.js";
+
+import { ApyMap, ChainId, FeesInterface, Usdc, VaultStatic } from "..";
+import { Context } from "../context";
+import { Yearn } from "../yearn";
+
+const subgraphFetchQueryMock = jest.fn();
+const visionApyMock: jest.Mock<Promise<ApyMap<string>>> = jest.fn();
+const lensAdaptersVaultsV2AssetsStaticMock: jest.Mock<Promise<VaultStatic[]>> = jest.fn();
+const oracleGetPriceUsdcMock: jest.Mock<Promise<Usdc>> = jest.fn();
+const getBlockNumberMock: jest.Mock<Promise<number>> = jest.fn();
+
+const dateMock = new Date("2022-01-01");
+jest.spyOn(global, "Date").mockImplementation(() => dateMock as unknown as string);
```

Comment:
nit: I think this is ok, but in `cache.spec.ts` I've used the following which seems a bit cleaner to me
```typescript
const nowMock = new Date(4242, 4, 2).getTime();
jest.spyOn(Date.prototype, "getTime").mockReturnValue(nowMock);
```
No action needed if you prefer as is